### ISSUE TITLE
twisted.internet.test.test_socket and twisted.internet.test.test_time need to be added to dist3.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8741]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8741|
|Type|enhancement|
|Created|2016-08-10 23:23:11Z|
|Branch|https://github.com/twisted/twisted/tree/8741-rodrigc-dist-py3|

twisted.internet.test.test_socket and twisted.internet.test.test_time need to be added to dist3.py so that they run under the Python 3 test environment in tox.

<details><summary>Searchable metadata</summary>

```
trac-id__8741 8741
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8741_rodrigc_dist_py3 8741-rodrigc-dist-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470871391088953 1470871391088953
changetime__1474198569965514 1474198569965514
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/483

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset a5d6577764ca0170934b06567c3cd6991b6f8ea5
```
#!CommitTicketReference repository="" revision="a5d6577764ca0170934b06567c3cd6991b6f8ea5"
Merge pull request #483 from twisted/8741-rodrigc-dist-py3

Author: rodrigc
Reviewer: alex
Fixes: #8741

Add test_socket and test_time to dist3.py
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing the review tag from closed tickets
